### PR TITLE
Merge styles of tables across the reports

### PR DESCRIPTION
**Pull request by dmydlo**
Hi @calellowitz @Rohit25negi,
I have fixed the discrepancies between tables styles across the reports.
Need QA: Yes
Environment: n/a
Locally Tested: Yes
Regards, Dawid

### CODE REVIEW

**Review by calellowitz — APPROVED** [review]
This seems fine, but why is this custom and using some shared styling with the rest of the reports?

File: custom/icds_reports/static/css/icds_dashboard.css [hunk 1]
```diff
@@ -874,23 +874,11 @@ hr {
     vertical-align: middle;
```
Comment (calellowitz) [inline custom/icds_reports/static/css/icds_dashboard.css]: Why is their custom text alignment in this report?

**Review by calellowitz — CHANGES_REQUESTED** [review]
Changing this to not approved until the questions are answered

File: custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ function ServiceDeliveryDashboardController($scope, $http, $location, $routePara
     }
 
     function renderHeaderTooltip(header, tooltipContent) {
-        return '<i class="fa fa-info-circle headerTooplip" style="float: right;" ><div class="headerTooplipText">' + tooltipContent + '</div></i><span>' + header + '</span>';
+        return '<i class="fa fa-info-circle headerTooltip" style="float: right;" ><div class="headerTooltipText">' + tooltipContent + '</div></i><span>' + header + '</span>';
```
Comment (Rohit25negi) [inline custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js]: Not a blocker,
But I think using [this](https://www.w3schools.com/js/js_htmldom_nodes.asp) for creating the html DOM would be much more readable and less prone to such errors.

**Review by Rohit25negi — APPROVED** [review]
Looks good to me

File: custom/icds_reports/static/css/icds_dashboard.css (hunk 1 above)
Comment (dmydlo) [reply custom/icds_reports/static/css/icds_dashboard.css]: Right, I am going to remove it.

**Comment by calellowitz** [comment]
@dmydlo 
Merging so we get this out but what does the remaining custom styling do, and why is it necessary?

**Review by calellowitz — APPROVED** [review]

File: custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js (hunk 2 above)
Comment (orangejenny) [reply custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js]: Agreed that concatenating HTML bits is hard to read. Another alternative is to use underscore templates [like this](https://github.com/dimagi/commcare-hq/blob/73f7f914761f93b4751bdc05d0520ccca49ae4eb/corehq/apps/data_interfaces/static/data_interfaces/js/find_by_id.js#L66-L68).

**Pull request merged**